packages/vue/src/components/molecules/SfScrollable/SfScrollable.js

```diff
@@ -0,0 +1,47 @@
+import simplebar from "simplebar-vue";
+import "simplebar/dist/simplebar.min.css";
+import SfButton from "../../atoms/SfButton/SfButton.vue";
+
+export default {
+  name: "SfScrollable",
+  components: {
+    simplebar,
+    SfButton
+  },
+  data() {
+    return {
+      isHidden: true
+    };
+  },
+  props: {
+    maxContentHeight: {
+      type: String,
+      default: ""
+    },
+    showText: {
+      type: String,
+      default: ""
+    },
+    closeText: {
+      type: String,
+      default: ""
+    }
+  },
+  mounted() {
+    this.$el.style.setProperty("--content-height", this.maxContentHeight);
+  },
+  methods: {
+    handleClick() {
+      this.isHidden = !this.isHidden;
+      if (!this.isHidden) {
+        this.$refs.content.$el.style.setProperty("max-height", "unset");
+        this.$refs.content.$el.style.setProperty("overflow-y", "unset");
+      } else {
+        this.$refs.content.$el.style.setProperty(
+          "max-height",
+          this.maxContentHeight
+        );
+      }
+    }
+  }
```

Comment:
Could you check if you can convert this logic to a `computed` property?
I think `handleClick` should only set `isHidden` (can even be done inside the template), the rest may work with only computed properties.

Reply:
Moved it to a watcher and `isHidden` is now handled inside the template

Reply:
Super cool 👍